test/e2e/tests/metamask-responsive-ui.spec.js

```diff
@@ -0,0 +1,233 @@
+        await driver.findElement({ text: 'Congratulations', tag: 'div' });
+        await driver.clickElement({
+          text: enLocaleMessages.endOfFlowMessage10.message,
+          tag: 'button',
+        });
+
+        // Show account information
+        // show account details dropdown menu
+        await driver.clickElement(
+          '[data-testid="account-options-menu-button"]',
+        );
+        const options = await driver.findElements(
+          '.account-options-menu .menu-item',
+        );
+        assert.equal(options.length, 3); // HD Wallet type does not have to show the Remove Account option
+        // click outside of menu to dismiss
+        // account menu button chosen because the menu never covers it.
+        await driver.clickPoint('.account-menu__icon', 0, 0);
+
+        // logs out of the vault
+        await driver.clickElement('.account-menu__icon');
+
+        const lockButton = await driver.findClickableElement(
+          '.account-menu__lock-button',
+        );
+        assert.equal(await lockButton.getText(), 'Lock');
+        await lockButton.click();
+      },
+    );
+  });
+
+  it('Importing existing wallet from login page', async function () {
+    const driverOptions = { responsive: true };
+    const testSeedPhrase =
+      'phrase upgrade clock rough situate wedding elder clever doctor stamp excess tent';
+
+    await withFixtures(
+      {
+        fixtures: 'imported-account',
+        driverOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        // Import seed phrase
+        const restoreSeedLink = await driver.findClickableElement(
+          '.unlock-page__link--import',
+        );
+        assert.equal(
+          await restoreSeedLink.getText(),
+          'Import using account seed phrase',
+        );
+        await restoreSeedLink.click();
+
+        await driver.clickElement('.import-account__checkbox-container');
+
+        await driver.fill('.import-account__secret-phrase', testSeedPhrase);
+
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.fill('#confirm-password', 'correct horse battery staple');
+        await driver.clickElement({
+          text: enLocaleMessages.restore.message,
+          tag: 'button',
+        });
+
+        // switches to localhost
+        await driver.clickElement('.network-display');
+        await driver.clickElement({
+          xpath: `//span[contains(@class, 'network-name-item') and contains(text(), 'Localhost 8545')]`,
+        });
+
+        // balance renders
+        const balance = await driver.findElement(
+          '[data-testid="eth-overview__primary-currency"]',
+        );
+        await driver.wait(until.elementTextMatches(balance, /100\s*ETH/u));
+
+        // logs out of the vault
+        await driver.clickElement('.account-menu__icon');
```

Comment:
Nit: This step of logging out of the vault doesn't seem necessary for testing that the onboarding worked.